src/azmi-main/blob/IBlobClient.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace azmi_main
 {
-    interface IBlobClient
+    public interface IBlobClient
```

Comment:
I could not leave it as private, as it is used in Execute method now, to inject testing